dataactbroker/handlers/fileHandler.py

```diff
@@ -954,18 +970,21 @@ def build_file_map(self, request_params, file_type_list, response_dict, upload_f
         """
         for file_type in file_type_list:
             # if file_type not included in request, skip it, checks for validity are done before calling this
-            if not request_params.get(file_type):
+            if not file_dict.get(file_type):
                 continue
-
-            file_name = request_params.get(file_type)
+            file = file_dict.get(file_type)
```

Comment:
Can we use a more descriptive name here than`file`?

Reply:
So, I struggled a bit with this because at this point in the code, that variable "file" may refer to either a) a string which is the filename, or b) the actual file binary... its not determined yet. The next few lines that follow this parse out whether its a string or a file. So I picked "file" because it kinda covers both, but I guess it could be something more specific... like "file_reference"? open to any other thoughts though...

Reply:
I think `file_reference` works. I just try to avoid `file` because it was a function in earlier versions of python.